.github/workflows/_mac-test.yml

```diff
@@ -142,7 +142,7 @@ jobs:
           export PR_BODY="${PR_BODY//[\'\"]}"
           arch
 
-          ${CONDA_RUN} python3 -mpip install dist/*.whl
+          ${CONDA_RUN} python3 -mpip install $(echo dist/*.whl)[opt-einsum]
```

Comment:
The echo is necessary because path expansion doesn't occur anymore with the addition of `[opt-einsum]`